src/Mvc/Mvc.RazorPages/src/Infrastructure/PageActionInvokerProvider.cs

```diff
@@ -52,7 +52,7 @@ internal class PageActionInvokerProvider : IActionInvokerProvider
             IModelBinderFactory modelBinderFactory,
             ITempDataDictionaryFactory tempDataFactory,
             IOptions<MvcOptions> mvcOptions,
-            IOptions<HtmlHelperOptions> htmlHelperOptions,
+            IOptions<MvcViewOptions> mvcViewOptions,
```

Comment:
Could we get an E2E test for this so we don't inadvertently break it? We would have caught that this didn't work the first time around had we written one

Reply:
Added a test to make you happy.